src/main/java/org/folio/qm/converter/impl/MarcHoldingsDtoConverter.java

```diff
@@ -20,4 +44,16 @@ protected String getExternalId(ParsedRecordDto source) {
   protected String getExternalHrId(ParsedRecordDto source) {
     return source.getExternalIdsHolder().getHoldingsHrid();
   }
+
+
+  @Override
+  protected Map<String, Object> splitGeneralInformationControlField(String content, String leader) {
+    List<ControlFieldItem> controlFieldItems = Arrays.asList(ACQ_STATUS, ACQ_METHOD, ACQ_ENDDATE, COMPL, COPIES,
```

Comment:
I think will be better to make this list a constant in the current class because in this case it will be initialized only once instead of each time record is converting.

Reply:
Same for MarcHoldingsQmConverter

Reply:
done